Soma-Technologies-Inc/visitAfrica-backend
# #175494169  Users should receive descriptive signup validation errors

### What does this PR do?
- signup/registration error message

### Description of Task to be completed?
- Users should receive descriptive signup/Registration validation errors

### How should this be manually tested?

- open the terminal in your text editor 

- run `https://github.com/Soma-Technologies-Inc/visitAfrica-backend.git` to clone the project 

- run `git checkout ft-signup-error-message-175494169` to navigate to my branch 

- run `npm install to install all the packages` 

- setup a .env file using .env.example format 

- run `npx sequelize db:migrate` 

- run `npm run startDev` to start the server 

- open your postman then make a request to this route of creating user POST /api/auth/signup and create a user 

- while creating the user,write some wrong inputs to check for validations and see feedback 

- try to register a user and re-register again with the same email address and see if you get feedback that email already exists or used before.

### Any background context you want to provide?

- [#175494169](https://www.pivotaltracker.com/story/show/175494169)

### Screenshots (if appropriate)
![Screenshot from 2020-12-05 09-26-14](https://user-images.githubusercontent.com/70438710/101236726-2b2ce680-36dc-11eb-8f4a-f14a669c9a84.png)
![Screenshot from 2020-12-06 12-13-42](https://user-images.githubusercontent.com/70438710/101277357-9221cc80-37bc-11eb-8238-fbc1ca3179d2.png)
